packages/integration-sdk-core/src/types/instance.ts

```diff
@@ -1,5 +1,7 @@
 export type IntegrationInstanceConfig = Record<string, any>;
 
+export type IntegrationExecutionConfig = Record<string, any>;
+
```

Comment:
I think we should consider making this interface an `object` because otherwise it can contain anything, but should be extended to define exactly what the integration will produce.

Reply:
Are you suggesting **a)**

```diff
- export type IntegrationExecutionConfig = Record<string, any>;
+ export type IntegrationExecutionConfig = object;
```
Or **b)**

```diff
- export type IntegrationExecutionConfig = Record<string, any>;
+ export type IntegrationExecutionConfig = Record<string, object>;
```

I think the former but I could use an expansion on your justification... an opportunity for learning on my part 😅

Reply:
great suggestion. Thanks.

Reply:
Done.